mkosi/__init__.py

```diff
@@ -4375,25 +4375,77 @@ def generate_key_cert_pair(args: Args) -> None:
     )
 
 
-def bump_image_version() -> None:
+def run_bump_script(config: Config, version: str) -> str:
```

Comment:
Having it close to the bump function makes sense, but all the other `run_foo_script` are together, so let's add it there. We've been wanting to make them share code for a long time anyway.